README.md

```diff
@@ -464,13 +464,13 @@ The two leading spaces in front of "# Hello" would be left-trimmed from all line
 
 ✅
 
-```md
+~~~md
```

Comment:
Why?

Reply:
There's a triple backtick inside of this example, which causes it to terminate prematurely. Using the alternate syntax for the wrapping codeblock allows this to display correctly:
~~~md
<div>
```js
var some = code();
```
</div>
~~~

instead of:
```md
<div>
```js
var some = code();
```
</div>
```